smoke_tests/README.md

```diff
@@ -28,6 +28,9 @@ yarn smokeTestCatalogueApiProd
 ## Running stress tests
 
 **Important:** Ensure that you are not going to put undue load on the production services (or dependent Elastic clusters)! Lack of due care could break the catalogue API!
+
+To stop a running test you can simply Ctrl-C t interrupt execution.
```

Comment:
```suggestion
To stop a running test you can Ctrl-C to interrupt execution.
```

nit: missing letter, don't say "simply" in documentation

Reply:
firmly?